superset-frontend/src/components/Select/styles.tsx

```diff
@@ -142,6 +142,8 @@ export const DEFAULT_STYLES: PartialStylesConfig = {
     return [
       provider,
       css`
+        border-width: ${isMulti ? '1px' : '0px'};
+        background-color: ${isMulti ? 'white' : '#F0F0F0'};
```

Comment:
these colors should probably be found within `supersetColors`, otherwise they should be added there